plugins/pur/core/src/test/resources/log4j2.xml

```diff
@@ -16,6 +16,7 @@
       <!-- Append messages to the console -->
       <!-- ============================== -->
       <Console name="PENTAHOCONSOLE">
+         <ThresholdFilter value="INFO"/>
```

Comment:
I looked at the history of the corresponding log4j.xml and it didn't have this threshold set.

https://github.com/pentaho/pentaho-kettle/blob/9.2.0.2/plugins/pur/core/src/test/resources/log4j.xml